lib/components/narrative/tabbed-itineraries.js

```diff
@@ -105,54 +111,69 @@ class TabButton extends Component {
   }
 
   render () {
-    const {index, isActive, itinerary, timeFormat} = this.props
-    const timeOptions = {
-      format: timeFormat,
-      offset: getTimeZoneOffset(itinerary)
-    }
+    const {currency, index, isActive, itinerary, timeFormat} = this.props
     const classNames = ['tab-button', 'clear-button-formatting']
     const { caloriesBurned } = calculatePhysicalActivity(itinerary)
     const {
-      centsToString,
       maxTNCFare,
       minTNCFare,
       transitFare
     } = calculateFares(itinerary)
     // TODO: support non-USD
     const minTotalFare = minTNCFare * 100 + transitFare
-    const plus = maxTNCFare && maxTNCFare > minTNCFare ? '+' : ''
     if (isActive) classNames.push('selected')
     return (
       <Button
         className={classNames.join(' ')}
         key={`tab-button-${index}`}
         onClick={this._onClick}
       >
-        <div className='title'><span>Option {index + 1}</span></div>
+        <div className='title'><span>
+          <FormattedMessage
+            id='components.TabbedItineraries.optionNumber'
+            values={{optionNum: index + 1}}
+          />
+        </span></div>
         <div className='details'>
           {/* The itinerary duration in hrs/mins */}
-          {formatDuration(itinerary.duration)}
+          <FormattedDuration duration={itinerary.duration} />
 
           {/* The duration as a time range */}
           <span>
             <br />
-            {formatTime(itinerary.startTime, timeOptions)} - {formatTime(itinerary.endTime, timeOptions)}
+            <FormattedTime
+              endTime={itinerary.endTime}
+              startTime={itinerary.startTime}
+              timeFormat={timeFormat}
+            />
           </span>
 
           {/* the fare / calories summary line */}
           <span>
             <br />
-            {minTotalFare ? <span>{`${centsToString(minTotalFare)}${plus}`} &bull; </span> : ''}
-            {Math.round(caloriesBurned)} Cal
+            {minTotalFare ? <span> <FormattedMessage
+              id='components.TabbedItineraries.fareCost'
+              values={{
+                minTotalFare: (
+                  <FormattedNumber
+                    currency={currency}
+                    currencyDisplay='narrowSymbol'
+                    style='currency'
+                    value={minTotalFare / 100}
+                  />
+                ),
+                useMaxFare: maxTNCFare && maxTNCFare > minTNCFare ? 'true' : 'false'
+              }}
+            /> &bull; </span> : ''}
+            {Math.round(caloriesBurned)} <FormattedMessage id='common.itineraryDescriptions.calories' />
```

Comment:
Should the number be a part of the message here?

Reply:
Good call, updated in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84